### PR TITLE
issue: issues/195 and issues/200 

### PR DESCRIPTION
**Pull request by sub2zero**
https://github.com/provectus/swiss-army-kube/issues/195
https://github.com/provectus/swiss-army-kube/issues/200

### CODE REVIEW

File: examples/argocd/README.md [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
 kubectl -n argocd port-forward svc/argocd-server  8080:80
 ```
 Now you can open http://127.0.0.1:8080 in a browser, the password for accessing ArgoCD UI is stored in AWS System Manager Paramstore, you can retrieve it by command:
 ``` bash
 aws --region <your-region> ssm get-parameter  --with-decryption --name /<your-cluster-name>/argocd/password | jq -r '.Parameter.Value' 
 ```
+
+### ArgoCD
+to get current password:
+for the first time use init password ```get secrets argocd-initial-admin-secret -n argocd -o jsonpath="{.data.password}" | base64 -D```
```
Comment (wallhided) [inline examples/argocd/README.md:21]: missing ```kubectl``` in command

File: examples/argocd/README.md [hunk 2]
```diff
@@ -1,21 +1,35 @@
...
+
+```bash
+kubectl get secret -n argocd argocd-secret -o json | \
+  jq '.data|to_entries|map({key, value:.value|@base64d})|from_entries'
+```
+
+to set a password:
+
+```bash
+kubectl patch secret -n argocd argocd-secret \
```
Comment (wallhided) [inline examples/argocd/README.md:33]: Does it work after setting a new password value immediately?

File: examples/argocd/variables.tf [hunk 3]
```diff
@@ -37,9 +61,48 @@ variable "domain_name" {
 variable "argocd" {
   default = {
     repository = "swiss-army-kube"
-    branch     = "master"
-    owner      = "provectus"
+    branch     = "testLab"
+    owner      = "sub2zero"
```
Comment (wallhided) [inline examples/argocd/variables.tf:65]: Should be removed. Please leave previous values.

**Review by wallhided — CHANGES_REQUESTED** [review]
I left some comments. Please take a look.

File: examples/argocd/README.md (hunk 2 above)
Comment (sub2zero) [reply examples/argocd/README.md:33]: yes, but it more like advice

File: examples/argocd/README.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 # About
 That example demonstrates how to configure the EKS cluster with the ArgoCD application. A general idea of the usage of ArgoCD is managing all Kubernetes resources with it. ArgoCD provides us with a way of implementing the GitOps methodology for Kubernetes applications.
+
+## Used modules
+- terraform-aws-modules/vpc/aws
+- terraform-aws-modules/eks/aws
+- github.com/provectus/sak-argocd, curentrul does not work with k8s with 1.22, need to update helm chart
```
Comment (wallhided) [inline examples/argocd/README.md:7]: typo here

**Review by wallhided — CHANGES_REQUESTED** [review]

File: examples/argocd/variables.tf [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+
+variable "single_nat" {
+  type        = bool
+  description = "Use single Nat gateway or separeta for all AZ"
+  default     = true
+}
+
 variable "cluster_name" {
-  default     = "swiss-army-kube"
+  default     = "swiss-army-kube-sub2zero"
```
Comment (wallhided) [inline examples/argocd/variables.tf:26]: should be reverted to previous value

**Review by wallhided — APPROVED** [review]
LGTM

**Pull request merged**